gimme

```diff
@@ -757,7 +757,7 @@ _to_goarch() {
 : "${GIMME_HOSTARCH:=$(_to_goarch "$(uname -m)")}"
 : "${GIMME_ENV_PREFIX:=${HOME}/.gimme/envs}"
 : "${GIMME_VERSION_PREFIX:=${HOME}/.gimme/versions}"
-: "${GIMME_TMP:=${TMPDIR:-/tmp}/gimme}"
+: "${GIMME_TMP:=${TMPDIR:-/tmp/$(whoami)}/gimme}"
```

Comment:
Would it make more sense to instead default to something like `$HOME/.gimme/tmp`? What do we end up using this temporary directory for?